### DISCUSSION TITLE
Welcome to Vocabsieve Discussions!

### DISCUSSION THREAD

**Discussion by 1over137** [FreeLanguageTools/vocabsieve#38] [Announcements]
Feel free to ask for help or clarifications here, or if you have any suggestions or ideas for improving the software, but is not concrete enough for a feature request or issue report.

**Comment by Ceynou** [comment]
I don't know what's your goal for this software, but I think most of yomichan or migaku's features would be a good addition, at least in concept

**Reply by 1over137** [reply]
Better open a new thread for this. What features are you talking about? I can add features, as long as it is reasonably easy to implement and maintain, and does not break existing workflows. Some tasks are better suited to other tools, however, like I'm not including a video player, because it's better to use mpv with mpvacious or asbplayer.
